addons/pos_sale/models/sale_order.py

```diff
@@ -34,6 +34,13 @@ def _compute_amount_unpaid(self):
             total_pos_paid = sum(sale_order.order_line.filtered(lambda l: not l.display_type).mapped('pos_order_line_ids.price_subtotal_incl'))
             sale_order.amount_unpaid = sale_order.amount_total - (total_invoice_paid + total_pos_paid)
 
+    @api.depends('invoice_ids.state', 'currency_id', 'amount_total')
```

Comment:
```suggestion
    @api.depends('pos_order_line_ids.price_subtotal_incl', 'pos_order_line_ids.down_payment_details')
```

I believe this list will be added to the list of compute dependencies, right?